packages/amplify_datastore/ios/Classes/types/hub/FlutterHubElement.swift

```diff
@@ -46,6 +46,39 @@ public struct FlutterHubElement {
         self.lastChangedAt = hubElement.lastChangedAt
         self.deleted = hubElement.deleted
     }
+
+    init(
+        hubElement: MutationEvent,
+        modelSchemaRegistry: FlutterSchemaRegistry,
+        customTypeSchemaRegistry: FlutterSchemaRegistry,
+        modelName: String
+    ) throws {
+        do {
+            let decodedModel = try ModelRegistry.decode(modelName: modelName, from: hubElement.json) as! FlutterSerializedModel
+            model = try decodedModel.toMap(
+                modelSchemaRegistry: modelSchemaRegistry,
+                customTypeSchemaRegistry: customTypeSchemaRegistry,
+                modelName: modelName
+            )
+            self.version = hubElement.version
+            let serializedData = model["serializedData"] as! [String: Any]
```

Comment:
```suggestion
            let serializedData = model["serializedData"] as? [String: Any] ?? [:]
```

Reply:
At this point, `model["serializedData"] as! [String: Any]` should always valid, as `try decodedModel.toMap()` should've completed without issues.

Reply:
You know that. I might even know that. But someone looking at this program for the first time doesn't know that, and the compiler doesn't know that. And those are the people that matter most.

Reply:
And this is a good point.